README.md

```diff
@@ -1,73 +1,48 @@
-
-----
-
-Azure AD provides the Active Directory Authentication Library, or ADAL, for iOS clients that need to access protected resources.  ADAL’s sole purpose in life is to make it easy for your app to get access tokens.  To demonstrate just how easy it is, here we’ll build a Objective C To-Do List application that:
-
--	Gets access tokens for calling the Azure AD Graph API using the [OAuth 2.0 authentication protocol](https://msdn.microsoft.com/library/azure/dn645545.aspx).
--	Searches a directory for users with a given alias.
-
-To build the complete working application, you’ll need to:
-
-1. Register your application with Azure AD.
-2. Install & Configure ADAL.
-3. Use ADAL to get tokens from Azure AD.
-
-To get started, [download the completed sample](https://github.com/AzureADQuickStarts/NativeClient-iOS/archive/complete.zip).  You'll also need an Azure AD tenant in which you can create users and register an application.  If you don't already have a tenant, [learn how to get one](active-directory-howto-tenant.md).
+# An iOS Microsoft Graph API application with Azure AD v2.0 using third party libraries
 
-## *1. Determine what your Redirect URI will be for iOS*
+## Scenario
 
-In order to securely launch your applications in certain SSO scenarios we require that you create a **Redirect URI** in a particular format. A Redirect URI is used to ensure that the tokens return to the correct application that asked for them.
+This app is a multi-tenant app meaning it can be used by any Azure AD tenant or Microsoft Account.  It demonstrates Azure AD's OAuth2 stanbdards compliance by demonstrating how a developer can use a third party library to build apps to connect with enterprise users and access their Azure + O365 data via the Microsoft Graph.  During the auth flow, end users will be required to sign in and consent to the permissions of the application, and in some cases may require an admin to consent to the app.  The majority of the logic in this sample shows how to auth an end user and make a basic call to the Microsoft Graph.
 
-The iOS format for a Redirect URI is:
+![Topology](./images/topology.png)
 
-```
-<app-scheme>://<bundle-id>
-```
-
-- 	**aap-scheme** - This is registered in your XCode project. It is how other applications can call you. You can find this under Info.plist -> URL types -> URL Identifier. You should create one if you don't already have one or more configured.
-- 	**bundle-id** - This is the Bundle Identifier found under "identity" un your project settings in XCode.
+## Steps to Run
 
-An example for this QuickStart code would be: ***msquickstart://com.microsoft.azureactivedirectory.samples.graph.QuickStart***
+1. Register your application
```

Comment:
This section uses bold, tick marks, and not marking stuff fairly inconsistently.